llvm/test/CodeGen/Xtensa/brcc.ll

```diff
@@ -1,10 +1,10 @@
 ; RUN: llc -march=xtensa < %s | FileCheck %s
 
-; CHECK-LABEL: brcc1:
+; CHECK-LABEL: brcc_sgt:
```

Comment:
Autogenerate check lines?

Reply:
It isn't, and -verify-machineinstrs is also missing

Reply:
-verify-machineinstrs should be missing. We should stop unconditionally adding it to every test, and only in cases specifically testing things that hit a verifier error

Reply:
It should never be hit, unless it is an XFAIL test.
From my experience, adding -verify-machineinstrs to llc tests saves a lot of time catching bugs that otherwise could be caught only when building large applications.
Machine verifier is effectively `assert(mir.isValid())`, I don't see a reason why you woulndn't want this assert when running tests.

I don't insist though (I would if I were a maintainer of this backend).

Reply:
I added "-verify-machineinstrs" argument to the brcc.ll test and generated check lines.